inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,139 @@
+#Pseudocode may look like actual code...please ignore...
```

Comment:
It's hard to distinguish the pseudocode from actual code. Would you be able to comment out the pseudocode to make it distinguishable?